packages/website/modules/zero/components/accordion/accordionSection.js

```diff
@@ -44,6 +45,15 @@ function AccordionSection({ active, toggle, toggleOnLoad, reportUID, slug, disab
     reportUID(uid);
   }, [reportUID, uid]);
 
+  useEffect(() => {
+    if (open && header) {
+      const headerLabel = header.props.children.find(child => {
```

Comment:
this is a litttttle funky - hard for me to tell whether it would be possible to use a `Ref` (or whether that would be cleaner) but curious if you considered that rather than searching through children? not a blocker, but figured it's worth raising for posterity